webclient/src/modules/interactive-map.js

```diff
@@ -1,51 +1,67 @@
-                    initLeaflet(_this);
+                const el_js = document.createElement("script");
+                el_js.src = "/* @echo app_prefix */js/mapbox/* @if target='release' */.min/* @endif */.js";
+                el_js.onload = () => {
                     resolve();
                 }
                 head.appendChild(el_js);
             });
         },
-        initMap: function(id) {
-            var map = L.map('interactive-map');
-            // Gesture handling currently only on mobile
-            if (window.matchMedia &&
-                window.matchMedia("only screen and (max-width: 480px)").matches) {
-                map.gestureHandling.enable();
-            }
-            console.log(map);
-            L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
-                attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
-            }).addTo(map);
-            
+        initMarker: function () {
+            const markerDiv = document.createElement('div');
+            const markerIcon = document.createElement('span');
+            markerIcon.style.backgroundImage = `url(/* @echo app_prefix */assets/map-marker_pin.png)`;
+            markerIcon.style.width = `25px`;
+            markerIcon.style.height = `36px`;
+            markerIcon.style.top = `-33px`;
+            markerIcon.style.left = `-12px`;
+            markerIcon.classList.add("marker")
+            markerDiv.appendChild(markerIcon);
+            const markerShadow = document.createElement('span');
+            markerShadow.style.backgroundImage = `url(/* @echo app_prefix */assets/map-marker_pin-shadow.png)`;
+            markerShadow.style.width = `38px`;
+            markerShadow.style.height = `24px`;
+            markerShadow.style.top = `-20px`;
+            markerShadow.style.left = `-12px`;
+            markerShadow.classList.add("marker")
+            markerDiv.appendChild(markerShadow);
+            return new mapboxgl.Marker({element:markerDiv});
+        },
+        initMap: function(container_id) {
+            mapboxgl.accessToken= 'pk.eyJ1IjoiY29tbWFuZGVyc3Rvcm0iLCJhIjoiY2t6ZGJyNDBoMDU2ZzJvcGN2eTg2cWtxaSJ9.PY6Drc3tYHGqSy0UVmVnCg'
+            const map= new mapboxgl.Map({
+                container: container_id,
+
+                // create the gl context with MSAA antialiasing, so custom layers are antialiasing.
+                // slower, but prettier and therefore worth it for our use case
+                antialias: true,
+
+                //preview of the following style is available at
+                // https://api.mapbox.com/styles/v1/commanderstorm/ckzdc14en003m14l9l8iqwotq.html?title=copy&access_token=pk.eyJ1IjoiY29tbWFuZGVyc3Rvcm0iLCJhIjoiY2t6ZGJyNDBoMDU2ZzJvcGN2eTg2cWtxaSJ9.PY6Drc3tYHGqSy0UVmVnCg&zoomwheel=true&fresh=true#16.78/48.264624/11.670726
+                style: 'mapbox://styles/commanderstorm/ckzdc14en003m14l9l8iqwotq',
```

Comment:
```suggestion
                style: 'mapbox://styles/commanderstorm/ckzdc14en003m14l9l8iqwotq?optimize=true',
                
                center: [11.5748, 48.1400],  // Approx Munich
                zoom: 11,  // Zoomed out so that the whole city is visible
```

This sets the default view to approx Munich (zoomed out), so that one still gets an idea where that location is relative to/within Munich, but it doesn't have this long world zoom-in which requests a lot of data not really relevant.

Also, there appears to be an option to remove unused features and improve perfomance (https://docs.mapbox.com/help/troubleshooting/mapbox-gl-js-performance/#remove-unused-features)